build failed on android

Building wolf for android fails with the following error:

```
failed to run custom build command for `wolfengine v3.0.3 (/Users/siminbadri/ALPHA/Wolves/WolfEngine/wolf)`

Caused by:
  process didn't exit successfully: `/Users/siminbadri/ALPHA/Wolves/WolfEngine/target/release/build/wolfengine-81941bbc9f675588/build-script-build` (exit status: 101)
  --- stdout
  cargo:rustc-link-search=native=/Users/siminbadri/ALPHA/Wolves/WolfEngine/wolf/sys/build/MinSizeRel
  cargo:rustc-link-lib=static=wolf_sys
  cargo:rustc-link-search=native=/Users/siminbadri/ALPHA/Wolves/WolfEngine/wolf/sys/build/MinSizeRel/_deps/lz4-build/
  cargo:rustc-link-lib=static=lz4
  cargo:rerun-if-changed=sys/compression/lz4.h
  cargo:rerun-if-changed=sys/compression/lz4.c

  --- stderr
  /Users/siminbadri/ALPHA/Wolves/WolfEngine/wolf/sys/wolf.h:45:10: fatal error: 'stdio.h' file not found
  /Users/siminbadri/ALPHA/Wolves/WolfEngine/wolf/sys/wolf.h:45:10: fatal error: 'stdio.h' file not found, err: true
  thread 'main' panicked at 'couldn't build bindings for sys/compression/lz4.h', wolf/build.rs:367:33
  note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
[2022-05-15T06:21:42Z INFO  cargo_ndk::cli] If the build failed due to a missing target, you can run this command:
[2022-05-15T06:21:42Z INFO  cargo_ndk::cli] 
[2022-05-15T06:21:42Z INFO  cargo_ndk::cli]     rustup target install armv7-linux-androideabi
```

--- a/wolf/build.rs
+++ b/wolf/build.rs
@@ -324,7 +324,7 @@ fn bindgens(p_current_dir_path_str: &str, p_target_os: &str) {
             std::env::var("ANDROID_NDK_HOME").expect("could not get ANDROID_NDK_HOME");
 
         let clang_include = format!(
-            "-I{}toolchains/llvm/prebuilt/{}/sysroot/usr/include",
+            "-I{}/toolchains/llvm/prebuilt/{}/sysroot/usr/include",
             android_ndk_home_env, ANDROID_NDK_OS_VARIANT
         );
         let clang_include_asm = format!("{}/arm-linux-androideabi", clang_include);

